### PR TITLE
Release 0.3.0 tweaks

### PR DESCRIPTION
**Pull request by jakirkham**
* Adds info about the release to the changelog.
* Updates `cloudpickle.__version__`.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+0.3.0
+=====
+
+* Import submodules accessed by pickled functions
+* Support recursive functions inside closures
+* Fix `ResourceWarnings` and `DeprecationWarnings`
+* Assume modules with `__file__` attribute are not dynamic
+
```
Comment (rgbkrk) [inline CHANGES.md]: D'oh!

File: cloudpickle/__init__.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 from cloudpickle.cloudpickle import *
 
-__version__ = '0.2.2'
+__version__ = '0.3.0'
```
Comment (rgbkrk) [inline cloudpickle/__init__.py]: Double d'oh.

**Comment by rgbkrk** [comment]
I'll... ship a patch release. It's clearly been a long time since I've shipped a pure python package.

**Comment by jakirkham** [comment]
np. Was wondering if it would be worth using versioneer for this package.

ref: https://github.com/warner/python-versioneer

**Pull request merged**